cloudy_mozdef/cloudformation/mozdef-instance.yml

```diff
@@ -71,6 +71,9 @@ Parameters:
   MozDefSQSQueueName:
     Type: String
     Description: The name of the generic SQS queue used to pickup events.
+  DomainName:
```

Comment:
A note : I was hoping to do this (in #871) by passing in a zone and having CloudFormation handle the route53 record. I may switch to that model down the road.

Reply:
I think assuming that roue53 exists in the customer acct is a big leap.

Reply:
I was thinking of making it conditional (if you provide a zone as a parameter, use it, otherwise expect the user to deal with names on their own)